src/organizations/components/DeleteOrgReasonsForm.tsx

```diff
@@ -102,16 +96,15 @@ function DeleteOrgReasonsForm({errors = {}}: ComponentProps) {
       )}
       <Form.Element
         label="How can we improve?"
-        errorMessage={errors?.suggestions ?? ''}
         className="element improvement-suggestion--input"
       >
         <TextArea
           className="improvement-suggestions-input"
           onChange={e => setSuggestions(e.target.value)}
-          size={ComponentSize.Medium}
           testID="improvement-suggestions-input"
           value={suggestions}
           placeholder="How can we improve?"
+          style={{height: '120px'}}
```

Comment:
why not add this to the className?

Reply:
Saw it used as this at some other place(s). Should I still move it to a css class?